### ISSUE TITLE
i dont know if i have exactly the same problem, this is my error

### ISSUE THREAD

**Issue by rmartin177** [gabotechs/react-stl-viewer#12]
i dont know if i have exactly the same problem, this is my error 
![image](https://user-images.githubusercontent.com/34422739/187806911-31a9d8f6-0629-43ec-8947-9c7382c5cdc8.png)
and this my package.json, i am using next.js without typescript and i have install three and react-stl-viewer

{
  "name": "example",
  "version": "0.1.0",
  "private": true,
  "scripts": {
    "dev": "next dev",
    "build": "next build",
    "start": "next start",
    "lint": "next lint"
  },
  "dependencies": {
    "next": "12.2.5",
    "react": "18.2.0",
    "react-dom": "18.2.0",
    "react-stl-viewer": "^2.1.2",
    "three": "^0.144.0"
  },
  "devDependencies": {
    "autoprefixer": "^10.4.8",
    "eslint": "8.23.0",
    "eslint-config-next": "12.2.5",
    "postcss": "^8.4.16",
    "tailwindcss": "^3.1.8"
  }
}

_Originally posted by @rmartin177 in https://github.com/gabotechs/react-stl-viewer/issues/3#issuecomment-1233584076_

**Comment by gabotechs** [comment]
I see that this is a issue with bundling one of the dependencies https://github.com/pmndrs/drei/issues/747

**Comment by gabotechs** [comment]
Solved in version 2.1.3

**Comment by gabotechs** [comment]
additionally, while testing I ran into this https://github.com/vercel/next.js/issues/7755

I needed to have this in my `next.config.js`
```js
module.exports = {
  webpack: (config, { isServer }) => {
    if (!isServer) {
      config.resolve.fallback.fs = false;
    }
    return config;
  },
}
```
